content/ch-gates/multiple-qubits-entangled-states.ipynb

```diff
@@ -2374,6 +2374,19 @@
     "plot_histogram(results)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The other three Bell's states are,\n",
+    "$$\n",
+    "|\\Phi^-\\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle - |11\\rangle)$$\n",
+    "$$|\\Psi^+\\rangle = \\frac{1}{\\sqrt{2}} (|01\\rangle + |10\\rangle)$$\n",
+    "$$|\\Psi^-\\rangle = \\frac{1}{\\sqrt{2}} (|01\\rangle - |10\\rangle)$$\n",
+    "\n",
+    "They are of fundamental importance to quantum mechanics."
```

Comment:
Perhaps we could add a reference to some external sources on why they are important? Otherwise it feels a bit like an empty statement, cause the textbook does not continue talking about them in spite of their fundamentality.

Reply:
Perhaps, I could link to the quantum teleportation and superdense coding sections?